src/NzbDrone.Core.Test/ParserTests/ParserFixture.cs

```diff
@@ -60,6 +60,56 @@ public void should_parse_movie_title(string postTitle, string title)
             Parser.Parser.ParseMovieTitle(postTitle, true).MovieTitle.Should().Be(title);
         }
 
+        [TestCase(
+            "L'hypothèse.du.tableau.volé.AKA.The.Hypothesis.of.the.Stolen.Painting.1978.1080p.CINET.WEB-DL.AAC2.0.x264-Cinefeel.mkv",
+            "L'hypothèse du tableau volé AKA The Hypothesis of the Stolen Painting",
+            new string[] { "L'hypothèse du tableau volé", "The Hypothesis of the Stolen Painting" })]
```

Comment:
Is there a slightly neater way to set up these test cases? I haven't used C# in ages so I'm a bit out of the loop!